templates/components/products/product-view.html

```diff
@@ -13,6 +13,11 @@
             Note that these image sizes are coupled to image sizes used in /assets/js/theme/common/product-details.js
             for variant/rule image replacement
         --}}
+        {{#if product.images.length '>' 1 }}
+        <span class="carousel-content-change-message aria-description--hidden" role="status">
```

Comment:
as this class used for logic but not for styles, can we add "js-" prefix to it as it is done for example for play/pause button?

Reply:
np